builder/openstack/step_stop_server.go

```diff
@@ -25,27 +25,44 @@ func (s *StepStopServer) Run(ctx context.Context, state multistep.StateBag) mult
 		return multistep.ActionHalt
 	}
 
-	ui.Say(fmt.Sprintf("Stopping server: %s ...", server.ID))
-	if err := startstop.Stop(client, server.ID).ExtractErr(); err != nil {
```

Comment:
I think maybe a better way to handle this would be to inspect the err created here. If it's a 409, consider that a valid exit state, and multistep.ActionContinue

Reply:
Thank you very much for your suggestions and feedback!

I have updated the PR with a more compact change that you suggested. I am still getting the error, I will take a look into it today.

Reply:
I will take a look tomorrow again, but if you could take a look at the updated version and confirm if it was me who caused the error in the gist, it would be very helpful - I cannot find the precise spot in the code where the error occurs, despite the trace. The error still shows up when I test locally (but I am able to use the image, because it happens after the image is created).

Reply:
Many thanks for your patience, Go is really something totally new for me.